scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -210,6 +210,26 @@ <h3 class="header">Institute Settings</h3>
               </div>
               <!-- End of Alamut settings -->
 
+              <!-- ClinVar API settings -->
+              <div class="row mt-5 d-flex align-items-center">
+                <fieldset><legend>ClinVar API submissions<a class="ms-2 text-decoration-none" href="https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/" target="_blank" rel="noopener">*</a></legend>
+                  <div class="row">
+                    <div class="col-6">
+                      {{ form.clinvar_key.label(class="control-label") }}
+                      {{ form.clinvar_key(class='form-control', value=institute.clinvar_key if institute.clinvar_key) }}
+                    </div>
+                    <div class="col-6">
+                      {{form.clinvar_emails.label(class="control-label",data_bs_toggle="tooltip", data_bs_placement="top", title="Email addresses to send variant verification email to. Only accepts current Scout user emails: exceptions to this can be made by a db admin. Please ask for support!")}}
```

Comment:
Mm, it is a little confusing since you dont get feedback on if they matched when saving. Perhaps at least check when the trying to add if it fits the current user list?

![Screenshot 2023-06-13 at 12 00 53](https://github.com/Clinical-Genomics/scout/assets/758570/e6e7e285-b3b0-46a0-b745-6d651e673815)